package.json

```diff
@@ -52,6 +52,7 @@
     "@craco/craco": "^7.0.0-alpha.0",
     "@trivago/prettier-plugin-sort-imports": "^3.2.0",
     "@types/react-resizable": "^1.7.4",
+    "@types/react-syntax-highlighter": "^13.5.2",
```

Comment:
just noticed that all versions have `^`, maybe it would be safer to have the version fixed